examples/seismic/acoustic/acoustic_time_update_nb.ipynb

```diff
@@ -0,0 +1,109 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Time update for *Cerjan* damped constant density acoustics with forward approximation for $\\frac{\\partial}{\\partial t}$\n",
+    "\n",
+    "We show a derivation for the time update expression used for the constant density acoustic solver. You can compare the end result of this derivation in the last equation line below with lines 58-59 in the file ```examples/seismic/acoustic/operators.py```."
```

Comment:
Maybe the line reference is a bit difficult to track if a change in operator code happens. Maybe we can think a different way to do this.

Reply:
will copy the lines as you suggested below